src/app/dashboard/_dashboard.scss

```diff
@@ -7,35 +8,114 @@ $paperCardBackgroundColor: #212121 !default;
 
 #dashboardContainer {
 	background-color: $dashboardContainerBackgroundColor;
+	height: 100%;
 
-	.userbar {
+	// typographic defaults
+	font-family: Roboto, sans-serif;
+	font-style: normal;
+	font-size: 1em;
+	line-height: 1.2em;
```

Comment:
you'll have to show me some examples (or I'll have to look into it this weekend).